drivers/blktap2.py

```diff
@@ -1615,7 +1617,6 @@ def _activate_locked(self, sr_uuid, vdi_uuid, options):
             driver_info = target.sr.srcmd.driver_info
```

Comment:
These few lines of code used to be conditional on `self.tap_wanted()` but now they're run unconditionally. Is that deliberate? Or an accidental side-effect of the way you've moved the beginning of the `try` block?

Reply:
well spotted, that's a bug

Reply:
I pushed this change a little too quickly... So fixed I guess this time. :)